backend/remote/backend_apply.go

```diff
@@ -102,6 +95,23 @@ func (b *Remote) opApply(stopCtx, cancelCtx context.Context, op *backend.Operati
 		))
 	}
 
+	if len(op.Targets) != 0 {
+		currentAPIVersion, err := version.NewVersion(b.client.RemoteAPIVersion())
```

Comment:
@radditude maybe we should include a comment here noting that `RemoteAPIVersion` returns an invalid version number (empty string) for versions before 2.3, to help a future reader understand why a parse error is treated the same as a version mismatch below...

Reply:
Good idea! Added 👍